src/linker/Linker.Dataflow/ReflectionMethodBodyScanner.cs

```diff
@@ -257,19 +251,16 @@ public override bool HandleCall (MethodBody callingMethodBody, MethodReference c
 			if (calledMethodDefinition == null)
 				return false;
 
-			_scopeStack.UpdateCurrentScopeInstructionOffset (operation.Offset);
-			var analysisContext = new AnalysisContext (
-				_scopeStack.CurrentScope.Origin,
-				ShouldEnableReflectionPatternReporting (),
-				_context);
-
 			DynamicallyAccessedMemberTypes returnValueDynamicallyAccessedMemberTypes = 0;
 
 			bool requiresDataFlowAnalysis = _context.Annotations.FlowAnnotations.RequiresDataFlowAnalysis (calledMethodDefinition);
 			returnValueDynamicallyAccessedMemberTypes = requiresDataFlowAnalysis ?
 				_context.Annotations.FlowAnnotations.GetReturnParameterAnnotation (calledMethodDefinition) : 0;
 
-			var handleCallAction = new HandleCallAction (_context, this, analysisContext, callingMethodDefinition);
+			_origin = _origin.WithInstructionOffset (operation.Offset);
+			var origin = _origin;
```

Comment:
Is there any reason why we copy the value here?, eventually there are calls using sometimes origin and two lines later _origin

Reply:
No good reason, I think it was left over from the way I mechanically did some of the changes. Fixed, thank you!